components/patient-tools/src/main/java/org/phenotips/tools/PhenotypeMappingService.java

```diff
@@ -226,13 +228,9 @@ private Object getMapping(String mappingName)
         DocumentReference mappingDoc = getMappingDocument();
         Object result = getMapping(mappingDoc, mappingName);
         if (result == null) {
-            Map<String, Object> mappings;
             try {
-                if (this.bridge.getDocumentContentForDefaultLanguage(mappingDoc).contains("{{velocity")) {
-                    mappings = parseVelocityMapping(mappingDoc);
-                } else {
-                    mappings = parseGroovyMapping(mappingDoc);
-                }
```

Comment:
Ok. I'm guessing that the phenotype preview also has to be backwards compatible.